test/test_tableau.cpp

```diff
@@ -250,6 +519,8 @@ TEST_F(TestTableau, CircuitTranslation) {
   qc.emplace_back(compOP);
 
   EXPECT_NO_THROW(tableau = cs::Tableau(qc););
+  EXPECT_NO_THROW(fullTableau = cs::Tableau(
+                      qc, 0, std::numeric_limits<std::size_t>::max(), true));
```

Comment:
```suggestion
  EXPECT_NO_THROW(fullTableau = cs::Tableau(
                      qc, true));
```
Should also just work, right?